packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.css

```diff
@@ -17,7 +17,37 @@
   display: flex;
   text-decoration: none;
   padding-right: 8px;
+}
+
+.dc-text {
+  text-decoration: none;
   color: var(--dc-primary-colour);
+  display: inline-block;
+  position: relative;
+}
+
+.dc-text::before {
+  content: " ";
+  position: absolute;
+  transform: scaleX(0);
+  width: 0;
+  height: 1px;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  transition: transform 0.25s ease-out;
+}
+
+a.dc-text:hover::before {
```

Comment:
Do you actually need the 'a' tag selector? Couldn't you just use the .dc-text class you've introduced?

Just an FYI in the future, try to keep your selectors as short as possible. It just helps with code readability and maintenance.